catroid/src/main/res/values/strings.xml

```diff
@@ -1979,6 +1979,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="brick_show_variable_aligned_centered">centered</string>
     <string name="brick_show_variable_aligned_left">left</string>
     <string name="brick_show_variable_aligned_right">right</string>
+    <string name="brick_context_dialog_pick_color">Pick Color</string>
```

Comment:
```suggestion
    <string name="brick_context_dialog_pick_color">Pick color</string>
```
Please use standard English capitalization rules at all places. 

Note: If you find places where we have missed it, please correct it. Also in Paintroid please.

Reply:
Fixed